quickstart/201-azfw-with-avzones/readme.md

```diff
@@ -0,0 +1,41 @@
+# Deploying Azure Firewall in Availability Zones
+
+This template deploys a test network environment with [Azure Firewall](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/firewall) in Availability Zones. The network has one virtual network (VNet) with three subnets: AzureFirewallSubnet, server_subnet, and jump_subnet. The server-subnet and jump_subnet subnet each have a single, two-core Windows Server virtual machine.
```

Comment:
In main.tf, the subnets are named: AzureFirewallSubnet, subnet-server, jump_subnet